### ISSUE TITLE
Android project comes with build error

### ISSUE THREAD

**Issue by hannesa2** [ViroCommunity/virocore#2]
I simply cloned the repository and run into this

<img width="1153" alt="image" src="https://user-images.githubusercontent.com/3314607/115868147-48b8dd80-a43c-11eb-97e4-43ab92e17aab.png">

**Comment by ktemby** [comment]
I haven't got this working in the community version yet, but sharing some notes from working toward my fork of the viromedia virocore in case it helps others.

After opening this in Android Studio (AS), I followed AS's recommendations including updating Gradle for the project, and pointing manually to the ndk in instance. I added a `local.properties` file at the top level of the project and entry called ndk.dir e.g. `ndk.dir=/Users/<username>/Library/Android/sdk/ndk/<ndk version>`.  That addressed the error show above. However it doesn't get us to a working build.

Later on I came across this issue https://developer.android.com/studio/releases/gradle-plugin#cmake-imported-targets 
   > More than one file was found with OS independent path 'lib/x86/libprebuilt.so'

I added this to build.gradle in viroar:
    packagingOptions {
          pickFirst '**/*.so'
      }

While that got past compile issues, I was then getting issues related to "cannot find symbol class "ARDeclarativeNode". I didn't get past those, but did find some other packaging options which might eventually allow the JNI file content to play nice. e.g. in viroar I changed above to: 
  packagingOptions {
    exclude 'lib/armeabi-v7a/libarcore_sdk_c.so'
    exclude 'lib/arm64-v8a/libarcore_sdk_c.so'
  }

For virocore I used: 
    packagingOptions {
       exclude 'lib/arm64-v8a/libBulletDynamics.so'
        exclude 'lib/armeabi-v7a/libBulletDynamics.so'
        exclude 'lib/armeabi-v7a/libgvr.so'
        exclude 'lib/arm64-v8a/libgvr.so'
        exclude 'lib/armeabi-v7a/libfreetyped.so'
        exclude 'lib/arm64-v8a/libfreetyped.so'
        exclude 'lib/armeabi-v7a/libBulletCollision.so'
        //exclude 'lib/arm64-v8a/libBulletCollision.so'
        exclude 'lib/arm64-v8a/libvrapi.so'
        exclude 'lib/armeabi-v7a/libvrapi.so'
        exclude 'lib/arm64-v8a/libLinearMath.so'
        exclude 'lib/armeabi-v7a/libLinearMath.so'
    }

In conclusion, work in progress.

**Comment by doranteseduardo** [comment]
> I simply cloned the repository and run into this
> 
> <img alt="image" width="1153" src="https://user-images.githubusercontent.com/3314607/115868147-48b8dd80-a43c-11eb-97e4-43ab92e17aab.png">

That occurs because you have not configured Android NDK, simply add the sdk.dir path in the **local.properties** file.

**Comment by hannesa2** [comment]
When someone merge my pull requests, then we can come closer to a state, where CI is able to build and such issues are complete obsolete !
